code/obj/machinery/door/door_timer.dm

```diff
@@ -169,18 +170,17 @@
 /obj/machinery/door_timer/process()
 	..()
 	if (src.timing)
-		if (!last_tick) last_tick = world.time
-		var/passed_time = round(max(round(world.time - last_tick),10) / 10)
+		if (!last_tick) last_tick = TIME
+		var/passed_time = round(max(round(TIME - last_tick), 10) / 10)
```

Comment:
Seems kind of odd to set and then check against itself, but appreciate this is you replacing like-for-like so whatever.